src/pudeuko/pudeuko_service.rs

```diff
@@ -0,0 +1,30 @@
+use super::domain::{Item, ItemList};
+use super::storage::Storage;
+
+pub struct PudeukoService {
```

Comment:
Dawaj serwis do domeny ;)

Reply:
Nie wiem czy to dobry pomysł, pogadamy jutro w pracy?